### PR TITLE
Ursinas first webpage

### PR DESCRIPTION
**Pull request by uursina**
I've created a webpage which is a blog about some of my favorite topics, interests, food etc.
I replaces the index.html file with a file called home.html.
I struggled most with css, as I initially created a separate css file vor each web page but later found out that I can just link the already existing css file if the page formation is the same.

### CODE REVIEW

File: code/home.html [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+<!--menu at the top-->
+    <menu>
+        <ul class="horizontal-menu">
+             <a href="home.html"> <i class="fa fa-envira" style="font-size:40px;color: #2f2e2e"></i></a>
+             <li> <a href="food.html"> food </a> </li>
+             <li> <a href="nature.html"> nature </a> </li>
+             <li> <a href="interests.html"> interests </a> </li>
+             <li> <a href="metime.html"> me time</a> </li>
+           </ul>
+    </menu>
```
Comment (HIPPIEKICK) [inline code/home.html:20]: Super nice with some navigation! Just a heads up - `nature.html` doesn't exist. Also, remember to indent your HTML when an element contains another element, like this:
```
<li>
  <a href="food.html"> food </a>
</li>
<li>
  <a href="nature.html"> nature </a>
</li>
<li>
  <a href="interests.html"> interests </a>
</li>
<li>
  <a href="metime.html"> me time</a>
</li>
```

File: code/home.html [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+</header>
+
+
+<body>
```
Comment (HIPPIEKICK) [inline code/home.html:24]: Woops, this body-tag ended up a little too far down. All your HTML content should be inside the `<body></body>`, so move this one up to line 10 so the beginning of your file looks like this:

```
<!DOCTYPE html>
<html lang="en">
<head>
    <meta charset="UTF-8">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Document</title>
    <link rel="stylesheet" type="text/css" href="home.css">
    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
</head>
<body>
    <header>
<!--menu at the top-->
      <menu>
```

File: code/home.html [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+    
+    <div class="body-standard">
```
Comment (HIPPIEKICK) [inline code/home.html:26]: There's an element called `main` that you could use for your main content to make your HTML more semantic

File: code/home.html [hunk 4]
```diff
@@ -0,0 +1,72 @@
...
+ <!--Introduction to the page-->       
+        <section class=intro>
```
Comment (HIPPIEKICK) [inline code/home.html:28]: Remember quotation marks when declaring classes:
`<section class="intro">`

File: code/home.html [hunk 5]
```diff
@@ -0,0 +1,72 @@
...
+                    <a href="books.html"> <img src="books.jpg" alt="books" style="width:100%"> </a>
+                </div>
+                <div class="column">
+                    <a href="brunch.html"> <img src="brunch.jpg" alt="brunch" style="width:100%"> </a>
+                </div>
+                <div class="column">
+                    <a href="coding.html"> <img src="codeexcursion.jpg" alt="coding" style="width:100%"> </a>
+                </div>
+              </div>
+        </figure>
```
Comment (HIPPIEKICK) [inline code/home.html:48]: Consider removing the `div` with class name `column` and apply that same styling on the `a`-tags directly (by adding the class name to that element instead). Maybe `row` could be called `"container"` and the `a`-tags could be called `"image-card"` or just `"card"` to make it more descriptive?

Comment (HIPPIEKICK) [reply code/home.html:48]: Also, try to keep all the styling in the CSS-file to be able to access it easier if you want to change something :)

File: code/books.css [hunk 6]
```diff
@@ -0,0 +1,83 @@
+@import url('https://fonts.googleapis.com/css2?family=Rajdhani:wght@300&family=Zilla+Slab:wght@500&display=swap');
+
+menu
+{
+  margin:0;
+  padding:2px;
+  background-color:white;
+}
```
Comment (HIPPIEKICK) [inline code/books.css:8]: The convention for formatting CSS is having the opening bracket on the same row as the element (with space in between), and to have space between the CSS property and its value, like this:

```
menu {
  margin: 0;
  padding: 2px;
  background-color: white;
}
```

File: code/books.css [hunk 7]
```diff
@@ -0,0 +1,83 @@
...
+  color: #2f2e2e;
+  font-size:40px;
+  font-family: 'Rajdhani', sans-serif;
+}
+menu .horizontal-menu
+{
+  list-style:none;
+  text-align: right;
+  padding-right: 100px;
+}
```
Comment (HIPPIEKICK) [inline code/books.css:20]: These are styling the same element, so you could put them together:
```
menu .horizontal-menu {
  color: #2f2e2e;
  font-size: 40px;
  font-family: 'Rajdhani', sans-serif;
  list-style: none;
  text-align: right;
  padding-right: 100px;
}
```

File: code/books.css (hunk 6 above)
Comment (HIPPIEKICK) [reply code/books.css:8]: This comment applies to all elements and CSS-files :)

File: code/books.css [hunk 8]
```diff
@@ -0,0 +1,83 @@
...
+  font-weight:bold;
+}
+
+body {
+        background-color: #F2F2F2;
+        position: absolute;
+        width: 100%;
+        height: 100%;
+        margin:0px 0px;
+}
```
Comment (HIPPIEKICK) [inline code/books.css:42]: Be consistent with the indentation. When the value is 0 you don't have to specify the unit:
```
body {
  background-color: #F2F2F2;
  position: absolute;
  width: 100%;
  height: 100%;
  margin: 0;
}
```

File: code/books.html [hunk 9]
```diff
@@ -0,0 +1,48 @@
...
+        </menu>
+</header>
+
+<body>
+    <div class="body-standard">
+    
+    <video autoplay loop muted>
+        <source src="books.mp4" type="video/mp4">
+        Your web browser does not support the video element.
+    </video>
```
Comment (HIPPIEKICK) [inline code/books.html:29]: Super nice with a video! 💯

File: code/books.html [hunk 10]
```diff
@@ -0,0 +1,48 @@
...
+    <p>What I like best are books that take place in a country or city I already know. This gives me the opportunity to return to this place and to get to know it through the eyes of another person - often in a different time period.</p>
+    </section>
+
+    <section class=subtext>
+    <h2>Public Bookcase</h2>
+    <p>When I go to a bookstore, I am often overwhelmed by the many choices. In addition, my limited student budget does not always allow me to buy a new book. Therefore I discovered the public bookcases for me. </p>
+    <p>What is a <a href="https://www.srf.ch/news/regional/aargau-solothurn/phaenomen-buecherschraenke-manchmal-versaufen-wir-fast-in-den-buechern">public bookcase</a>?</p>
+    <p>Here you can find a <a href="https://www.e-chline-schritt.ch/oeffentlicher-buecherschrank/">list</a> of public bookcases in Switzerland, and information about the <a href="https://www.bern.ch/mediencenter/medienmitteilungen/aktuell_ptk/offene-buecherschraenke-neu-in-neun-berner-gruenanlagen">project</a> in Bern. </p>
+    </section>
```
Comment (HIPPIEKICK) [inline code/books.html:42]: Missing quotation marks around the class name, and missing indentation ;)

File: code/brunch.html [hunk 11]
```diff
@@ -0,0 +1,46 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" type="text/css" href="brunch.css">
```
Comment (HIPPIEKICK) [inline code/brunch.html:7]: `brunch.css` does not exist. Did you mean to link `home.css`?

File: code/home.html [hunk 12]
```diff
@@ -0,0 +1,72 @@
...
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" type="text/css" href="home.css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+</head>
+<header>
+<!--menu at the top-->
+    <menu>
+        <ul class="horizontal-menu">
+             <a href="home.html"> <i class="fa fa-envira" style="font-size:40px;color: #2f2e2e"></i></a>
```
Comment (HIPPIEKICK) [inline code/home.html:14]: Yey, nice icon 🍃

File: code/food.html [hunk 13]
```diff
@@ -0,0 +1,43 @@
...
+             <li> <a href="nature.html"> nature </a> </li>
+             <li> <a href="interests.html"> interests </a> </li>
+             <li> <a href="metime.html"> me time</a> </li>
+           </ul>
+    </menu>
+</header>
+<body>
+    <div class="boxes">
+        <div class="content-box has-featured-img">
+          <img class="featured-img" src="coffee.jpg" alt="coffee">
```
Comment (HIPPIEKICK) [inline code/food.html:24]: When an HTML element has several attributes, it's good for readability to give them new lines:
```
<img
  class="featured-img"
  src="coffee.jpg"
  alt="coffee">
```

File: code/metime.html [hunk 14]
```diff
@@ -0,0 +1,44 @@
...
+<body>
+    <div class="boxes">
+        <div class="content-box has-featured-img">
+          <img class="featured-img" src="books.jpg" alt="book">
+          <h2 class="title">Getting Lost in a Book</h2>
+          <div class="content">
+            <p>Finding time to read a good book gives me the opportunity to let the daily sometimes busy life behind and <a href="books.html">...</a></p>
+          </div>
+        </div>  
+    </div>
```
Comment (HIPPIEKICK) [inline code/metime.html:31]: I really like the look of these boxes 🤩

**Review by HIPPIEKICK — COMMENTED** [review]
Great work Ursina, I really like the styling of your page, it looks super good. Most of my comments are about the formatting of your code. If you need any guidance regarding that, take a look [here](https://www.notion.so/technigo/Guidelines-for-how-to-write-good-code-69ffb59eac4e4358a7c5752efb873c4e). Most comments are applicable to several elements and several files (since they are similar).

- Double check so that you have linked a CSS-file in every HTML-file (some CSS-file is not existing).
- As you wrote, some of your CSS is the same for multiple files so you could have one CSS-file and link to that in every HTML-file to avoid having to make the same change in multiple files.

Be proud and keep up the good work! ⭐ 

/Matilda
Technigo

**Comment by uursina** [comment]
Thank you very much Matilda for your feedback! it is very helpful.

best regards,
Ursina

**Pull request closed without merging**